src/x86_64/def64.h

```diff
@@ -13,6 +13,15 @@ typedef __uint128_t u128;
 typedef u64 word;
 typedef u64 bytes;
 
+// XXX verify
```

Comment:
strictly speaking these are true regardless of how many bits wide the registers are